src/models/actions/CodeRun.spec.js

```diff
@@ -64,7 +64,9 @@ describe("CodeRun", () => {
 
     expect($result.actionReports.length).toBe(1);
     expect($result.actionReports[0].success).toBe(false);
-    expect($result.actionReports[0].shortSummary).toBe(`Code must return an object. Instead got string "Fuck off"`);
+    expect($result.actionReports[0].shortSummary).toBe(
+      `Error: Code must return an object. Instead got string \"Fuck off\"`
```

Comment:
```suggestion
      `Code must return an object. Instead got string \"Fuck off\"`
```

Reply:
And if you make this change, then we need to chang the eval strings in this test file too like this

```
code: `module.exports = (context) => {
                throw new Error("Fuck off")
            }`,
            ```